src/app/component/MissionDetailsCard.jsx

```diff
@@ -0,0 +1,384 @@
+      content: [{ text: fundedStatus }],
+    },
+  ];
+
+  const pickUpDetails = [
+    {
+      icon: LocationOnIcon,
+      content: [{ text: pickUpLocation }],
+    },
+    {
+      icon: ScheduleIcon,
+      content: [{ text: pickUpWindow }],
+    },
+  ];
+
+  const deliveryDetails = [
+    {
+      icon: LocationOnIcon,
+      content: [{ text: deliveryLocation }],
+    },
+    {
+      icon: ScheduleIcon,
+      content: [{ text: deliveryWindow }],
+    },
+    {
+      icon: PersonIcon,
+      content: [
+        { text: recipientName },
+        {
+          text: recipientPhoneNumber,
+          style: {
+            fontWeight: 600,
+            textDecoration: "underline",
+          },
+        },
+      ],
+    },
+  ];
+
+  return (
+    <>
+      <Grid className={classes.content} direction="column" container>
+        {showBackIcon && (
+          <Grid align="left" item>
+            <ArrowBackIcon
+              align="left"
+              className={classes.goBackIcon}
+              onClick={() => history.goBack()}
+            />
+          </Grid>
+        )}
+        <Grid>
+          <Card align="left">
+            <CardHeader
+              title={title}
+              titleTypographyProps={{ variant: "h3", component: "span", color: "textPrimary" }}
+              subheader={
+                <MissionDetailsIconList
+                  outerClass={classes.subheader}
+                  contentItems={subheaderItems}
+                  classes={classes}
+                />
+              }
+              className={classes.cardHeader}
+            />
+            <CardMedia image={cameraImage} title="Mission image" className={classes.image} />
+            <CardContent className={classes.cardContent}>
+              <MissionDetailsContent description={description} classes={classes} />
+              <MissionDetailsPickUpDeliveryHeader header="Pick Up Details" classes={classes} />
+              <MissionDetailsIconList
+                outerClass={classes.deliveryDetails}
+                contentItems={pickUpDetails}
+                classes={classes}
+              />
+              <MissionDetailsPickUpDeliveryHeader header="Delivery Details" classes={classes} />
+              <MissionDetailsIconList
+                outerClass={classes.deliveryDetails}
+                contentItems={deliveryDetails}
+                classes={classes}
+              />
+            </CardContent>
+            <Divider />
+            <CardActions>
+              <MissionDetailsButton
+                type={mission.status}
+                volunteerForMission={() => volunteerForMission(mission.id)}
+                startMission={() => startMission(mission.id)}
+                markMissionAsDelivered={() => markMissionAsDelivered(mission.id)}
+              />
+            </CardActions>
+            <CardActions>
+              <Grid container justify="center">
+                {mission.status === MissionStatus["tentative"] ||
```

Comment:
Can we extract this conditional rendering into a named function which returns the component based on the condition? Its easier to read and maintain.